spec/support/oidc_auth_helper.rb

```diff
@@ -72,6 +86,40 @@ def ial2_params(prompt: nil,
     ial2_params
   end
 
+  def ial2_strict_params(prompt: nil,
+                         state: SecureRandom.hex,
+                         nonce: SecureRandom.hex,
+                         client_id: OIDC_ISSUER)
+    ial2_strict_params = {
+      client_id: client_id,
+      response_type: 'code',
+      acr_values: Saml::Idp::Constants::IAL2_STRICT_AUTHN_CONTEXT_CLASSREF,
+      scope: 'openid email profile:name social_security_number',
+      redirect_uri: 'http://localhost:7654/auth/result',
+      state: state,
+      nonce: nonce,
+    }
+    ial2_strict_params[:prompt] = prompt if prompt
+    ial2_strict_params
```

Comment:
This is nearly identical to `ial2_params` ... can we just reuse + override?

```suggestion    
    ial2_params(prompt: prompt, state: state, nonce: nonce, client_id: client_id).
      merge(acr_values: Saml::Idp::Constants::IAL2_STRICT_AUTHN_CONTEXT_CLASSREF)
```